_posts/2020-08-12-DDoS-criminal-hackers.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "Documents Confirm DHS is Investigating Journalists as 'Criminal Hackers'"
+image: images/blogimages/ddos.png
+author: lucyparsonslabs
+teaser: "Documents obtained by LPL appear to confirm DHS is investigating journalists for their reporting"
+tags:
+  - First Amendment
+---
+
+Documents obtained by Lucy Parsons Labs under the Illinois Freedom of Information Act (FOIA)
+appear to confirm the US government is aggressively investigating a journalism
+non-profit as "criminal hackers" for publishing original reporting.
+
+The documents, first obtained and published by the transparency non-profit
+Distributed Denial of Secrets (DDoS), provide an intimate look into police
+agencies and fusion centers across the country. Established after 9/11 to
+disrupt terrorism plots, fusion centers have instead been challenged for massive
+privacy and civil liberties violations. DDoS obtained over 270 gigabytes of data from
+fusion centers and police departments and published the release in what it
+billed the #BlueLeaks release.
+
+Journalists and researchers immediately seized on the Blue Leaks release to
+reveal that police were obsessed with left-wing movements and ignoring
+far-right [domestic extremism](https://theintercept.com/2020/07/15/george-floyd-protests-police-far-right-antifa/).
+Blue Leaks further revealed that law enforcement inflated threats after the
+police murder of George Floyd in Minneapolis with some police stating, without evidence, that "Antifa"
+would carry out [car bombings](https://theintercept.com/2020/06/26/blueleaks-minneapolis-police-protest-fears/).
+The shoddy intelligence of fusion centers has been [well documented](https://www.hsgac.senate.gov/imo/media/doc/10-3-2012%20PSI%20STAFF%20REPORT%20re%20FUSION%20CENTERS.2.pdf)
+for over fifteen years.  
```

Comment:
I just dropped the "fifteen years". Technically the link goes to 2012, the most comprehensive look but its been ever since the centers got established. The time range isn't critical to the point of the sentence.